mail/fdm/Makefile

```diff
@@ -47,13 +50,7 @@ MAKE_FLAGS += \
 	PREFIX="/usr" \
 	$(if $(CONFIG_FDM_WITH_PCRE),PCRE=1)
 
-define Build/Prepare
-	$(call Build/Prepare/Default)
-	$(CP) ./src/compat/* $(PKG_BUILD_DIR)/
-endef
-
 define Build/Configure
-	( cd $(PKG_BUILD_DIR); ./autogen.sh )
 	$(call Build/Configure/Default)
```

Comment:
After your change, the whole Build/Configure section looks unnecessary, as it only contains a call to the default Configure.